org.bluesabre.MenuLibre.yaml

```diff
@@ -11,8 +11,8 @@ finish-args:
   - '--socket=fallback-x11'
   - '--socket=wayland'
   - '--share=ipc'
-  - '--filesystem=xdg-data/applications:create'
-  - '--filesystem=xdg-data/flatpak:ro'
+  # - '--filesystem=xdg-data/applications:create'
+  # - '--filesystem=xdg-data/flatpak:ro'
```

Comment:
the question here is: weren't these sanbox holes needed for some features?

Reply:
Yes, to create new desktop entries.

Reply:
https://github.com/flathub/flatpak-builder-lint/pull/146